lms/static/scripts/postmessage_json_rpc/server.js

```diff
@@ -101,6 +103,12 @@ export class Server {
       return;
     }
 
+    if (this._isJSONRPCNotificationRequest(event)) {
```

Comment:
I would be inclined to refactor this method and the one above so that they just take the event data as an argument, rather than the whole event, since they only use the `data` property. They don't even need to be methods, they can just be helper functions.